widgets/src/components/TaskCompletionForm/TaskCompletionFormConfig.js

```diff
@@ -0,0 +1,202 @@
+/* eslint-disable no-console */
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  FormGroup,
+  ControlLabel,
+  Button,
+  HelpBlock,
+  Row,
+  Col,
+  FormControl,
+} from 'patternfly-react';
+
+import { getConnections } from 'api/pda/connections';
+import { getProcesses } from 'api/pda/processes';
+import { getPageWidget, putPageWidget } from 'api/app-builder/pages';
+
+import 'patternfly-react/dist/css/patternfly-react.css';
+import 'patternfly/dist/css/patternfly.css';
+import 'patternfly/dist/css/patternfly-additions.css';
+
+class CompletionFormConfig extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      sourceList: [],
+      processList: [],
+      settings: {
+        uiSchema: '{}',
+      },
+      knowledgeSource: '',
+      selectedProcess: '',
+    };
+
+    this.onChangeKnowledgeSource = this.onChangeKnowledgeSource.bind(this);
```

Comment:
lets use arrow functions instead of `bind` to keep the standard

Reply:
I would prefer using normal class properties approach, we have used this in MFE for blueprint widgets